src/FuseWorker.js

```diff
@@ -1,17 +1,17 @@
-var channels = [];
-var npx = 0;
+const channels = [];
+let npx = 0;
```

Comment:
Does this file still have work left to do before changing the file extension to .ts?